phpstan.neon

```diff
@@ -9,5 +9,6 @@ parameters:
         # Some parent constructors are explicitly to be ignored
         - '#does not call parent constructor#'
         - '#Access to an undefined property PhpParser\\Node\\Param::\$isOptional#'
+        - '#Access to an undefined property PhpParser\\Node\\Expr::\$value#'
```

Comment:
Seems invalid: is a constant an `Expr`?

Reply:
Removed in favour of inline `@var` annotations.